### PR TITLE
fix windows builds for pip

### PR DESCRIPTION
**Pull request by arun3688**
### Purpose
This PR fixes windows build , for uploading the package to `PIP` installer

### Approach
Set environment variable for visual studio versions and display error messages to user

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -76,30 +76,30 @@ def run(self):
...
+        visualStudioVersion = ["VS15-Win64", "VS15-Win32", "VS14-Win64", "VS14-Win32"]
+        args = "";
+        if os.environ.get('VS15-Win64'):
+          args = visualStudioVersion[0]
+        elif os.environ.get('VS15-Win32'):
+          args = visualStudioVersion[1]
+        elif os.environ.get('VS14-Win64'):
+          args = visualStudioVersion[2]
+        elif os.environ.get('VS14-Win32'):
+          args = visualStudioVersion[3]
```
Comment (lochel) [inline setup.py:88]: ❓

Comment (lochel) [reply setup.py:88]: Make it a loop and avoid defining the keys a redundant way.

File: setup.py [hunk 2]
```diff
@@ -76,30 +76,30 @@ def run(self):
...
+        
+        if args:
+          subprocess.call(['configWinVS.bat', args])
+          subprocess.call(['buildWinVS.bat', args, '-j'+str(multiprocessing.cpu_count())])
+        else:
+          print("Environment Variable not set for Visual Studio version, Please set one of the following")
+          print("1. Variable name: VS15-Win64, ", "Variable value: path to vcvarsall.bat, ", "for Visual Studio 2017 64 bit")
+          print("2. Variable name: VS15-Win32, ", "Variable value: path to vcvarsall.bat, ", "for Visual Studio 2017 32 bit")
+          print("3. Variable name: VS14-Win64, ", "Variable value: path to vcvarsall.bat, ", "for Visual Studio 2015 64 bit")
+          print("4. Variable name: VS14-Win32, ", "Variable value: path to vcvarsall.bat, ", "for Visual Studio 2015 32 bit")
```
Comment (lochel) [inline setup.py:98]: Put the keys and description in a dict and the populate the message by iterating the dictionary. (Use the same dictionary to detect the vs version.)

**Review by lochel — CHANGES_REQUESTED** [review]

File: setup.py (hunk 2 above)
Comment (arun3688) [reply setup.py]: this is fixed  with https://github.com/OpenModelica/OMSimulator-pip/pull/2/commits/3052bb31c1bf13c5d04b98ce4b15b4109144f304

File: setup.py (hunk 1 above)
Comment (arun3688) [reply setup.py]: This is fixed with https://github.com/OpenModelica/OMSimulator-pip/pull/2/commits/3052bb31c1bf13c5d04b98ce4b15b4109144f304

**Review by lochel — APPROVED** [review]

**Pull request merged**